spec/features/indexing_custom_component_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Indexing Custom Component', type: :feature do
+  let(:components) do # an Array<Arclight::CustomComponent>
+    options = { component: Arclight::CustomComponent }
+    ENV['SOLR_URL'] = 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
Totally understand.  In the meantime, can we set the `SOLR_URL` to be the value from our configured solr URL? (which I believe is available to us through the global `Blacklight` object)